src/plugins/organisation/PMRWindow/src/pmrwindowwindow.cpp

```diff
@@ -182,113 +182,22 @@ void PmrWindowWindow::busy(const bool &pBusy)
-    } else {
-        finished();
-    }
+    QMessageBox::information(Core::mainWindow(), pWhere, pMessage, QMessageBox::Ok);
 }
 
 //==============================================================================
 
-void PmrWindowWindow::doCloneWorkspace(const QString &pWorkspace)
+void PmrWindowWindow::showInformation(const QString &pMessage)
 {
-    // Retrieve the name of an empty directory
-
-    QString dirName = Core::getExistingDirectory(tr("Select Empty Directory"),
-                                                 QString(), true);
-
-    if (!dirName.isEmpty()) {
-        // We have got a directory name, so before cloning the workspace, make
-        // sure that the dialog box has gone (indeed, otherwise, on OS X at
-        // least, we will see a blank area where the dialog box used to be)
-        // Note: this is a temporary solution until we implement issue #848...
-
-        QCoreApplication::processEvents();
-
-        // We have got a directory name where we can clone the workspace, so
-        // clone it
-
-        git_libgit2_init();
-
-        git_repository *gitRepository = 0;
-        QByteArray workspaceByteArray = pWorkspace.toUtf8();
-        QByteArray dirNameByteArray = dirName.toUtf8();
-
-        int res = git_clone(&gitRepository, workspaceByteArray.constData(),
-                            dirNameByteArray.constData(), 0);
-
-        if (res) {
-            const git_error *gitError = giterr_last();
-
-            QMessageBox::warning(Core::mainWindow(), tr("Clone Workspace"),
-                                 gitError?
-                                     tr("Error %1: %2.").arg(QString::number(gitError->klass),
-                                                             Core::formatMessage(gitError->message)):
-                                     tr("An error occurred while trying to clone the workspace."),
-                                 QMessageBox::Ok);
-        } else if (gitRepository) {
-            git_repository_free(gitRepository);
-        }
-
-        git_libgit2_shutdown();
-    }
-}
+    // Show a message box with informative text
 
-//==============================================================================
-
-void PmrWindowWindow::doShowExposureFiles(const QString &pExposureUrl)
-{
-    // Show the exposure files, but only if there are some
-
-    if (!mExposureFileNames.values(pExposureUrl).isEmpty()) {
-        mPmrWidget->showExposureFiles(pExposureUrl);
-    } else {
-        QMessageBox::information( Core::mainWindow(), windowTitle(),
-                                  tr("No exposure file URL could be found for <a href=\"%1\">%2</a>.").arg(pExposureUrl, mExposureNames.value(pExposureUrl))
-                                 +"<br/><br/>"+mInformationNoteMessage,
-                                  QMessageBox::Ok);
-    }
+    QMessageBox::information(Core::mainWindow(), windowTitle(),
+                             pMessage+"<br/><br/>"+mInformationNoteMessage,
```

Comment:
```mInformationNoteMessage``` really shouldn't be in ```PmrWindowWindow``` anymore. A slot like this one should only display the message it is given. In other words, ```mInformationNoteMessage ``` should now be part of